Lib/glyphsLib/types.py

```diff
@@ -281,15 +281,29 @@ def strftime(self, val):
 
 
 def parse_color(src=None):
+    # type: (Optional[str]) -> Optional[Union[Tuple[int, ...], int]]
+    """Parse a string representing a color value.
+    
+    Color is either a fixed color (when coloring something from the UI, see
+    the GLYPHS_COLORS constant) or a list of the format [u8, u8, u8, 1],
+    negative numbers are abs()'d. Glyphs does not support an alpha channel
+    as of 2.5.1 (confirmed by Georg Seifert), and always writes a 1 to it.
+    This is probably wrong, so we handle it like the other RGB values.
+    """
     if src is None:
         return None
+
+    # Tuple. Also handle cases like () and (1) like Glyphs.
     if src[0] == "(":
-        src = src[1:-1]
-        color = src.split(",")
-        color = tuple([int(c) for c in color])
-    else:
-        color = int(src)
-    return color
+        rgb = src[1:-1].split(",")
+        color_template = [0, 0, 0, 255]
+        for i, v in enumerate(rgb):
+            if v:  # Could be ""
```

Comment:
are you sure that one of the items in the color tuple in a `.glyphs` source file could be ""?
I could not verify this. When I set a glyphs' colorObject from the Macro Panel and save, I always get a tuple of four integers in range 0..255 (with exception of the last which is always 1, but Georg will fix that).